vllm/config.py

```diff
@@ -1540,6 +1540,11 @@ class LoadConfig:
     use_tqdm_on_load: bool = True
     """Whether to enable tqdm for showing progress bar when loading model
     weights."""
+    pt_load_map_location: Union[str, dict[str, str]] = "cpu"
+    """
+    pt_load_map_location: the map location for loading pytorch checkpoint, to
+    support loading checkpoints can only be loaded on certain devices like cuda.
+    Default to cpu."""
```

Comment:
Can you add a reference of an example or a link to torch.load's map_location doc? It isn't clear what format it should be

Reply:
Good point, also {"": "cuda"} seems not a good example, usually it's "cpu":"cuda", something like this.

Reply:
I thought Union of dict and str was not possible due to current parsing logic but let me try again to see if it can be fixed